### ISSUE TITLE
Grand Xmas Wish-List of FoxGuard Features

### ISSUE THREAD

**Issue by Inscrutable** [FoxDenStudio/FoxGuard#57]
Here is my wish list for FoxGuard features, that might get implemented in-the-fullness-of-time:

- [ ] (1) Region Entry / Exit messages (per handler, or handler group)
- [ ] (2) User self-service within admin-defined limits (# blocks, in designated super-region, prevent overlap on other people's regions, perms to set flags/delete) - Yeah, this is already on the RoadMap, but more details of the way it would/could/should work will help.
- [ ] (3) region sale or rent (-> economy API? or per econ service?) with ownership reversion on expiry 
- [ ] (4) a way to alter a region's boundaries
- [ ] (5) minor buff/debuff flags (eg heal, feed, harm, starve, etc) 
- [ ] (6) a handler to prevent certain (configurable) commands being executed in regions (eg tp, warp, gamemode, spawn, etc)
- [ ] (7) ability to allow explosions to cause injury but not break blocks (except the TNT, in that case)
- [ ] (8) fire handler - allow/deny fire spread (and block burning) by region
- [ ] (9) Death counter for people who expire in a region (with stat tracking?)
- [x] (10) Oatmeal with FLARD
- [ ] (11) more neat wands - eg measurement, "here", long-distance pos wand, teleport wand, debug wand, ... 
- [ ] (13)  fg docs as in-game books (or a wand)
- [ ] (14) dynmap integration or add-on to show configurably-flagged fg regions on forge-dynmap 
- [ ] (15) a handler to keep schematics assigned to a region and paste them in under configurable circumstances (or hook into edifice?)
- [ ] (16) some way to time how long a region has been vacant
- [ ] (17) Apocalypse Handler
- [ ] (18) Configurable snow drift / ice formation
- [ ] (19) Is there a direct way to tp to a region without having to look up co-ords?
- [x] (20) Hi foxy :tropical_fish:

**Comment by WillPowerCraft** [comment]
20 :D
5 and 6 would be pretty nice
7 allready working with TNT flowers :3

**Comment by Chakratos** [comment]
3 for sure!

**Comment by jaktari** [comment]
I want 3 for sure, 14 would be nice too( but 3 is something i really need for my server)
Thank you!

**Comment by vectrixdevelops** [comment]
To add to that also stop hunger in invincibility flag.

**Comment by gravityfox** [comment]
I might make hunger prevention it's own flag. It has it's own event for sure.

**Comment by vectrixdevelops** [comment]
Ok cool. 😄

**Comment by Inscrutable** [comment]
Good call. May I humbly suggest the flag ``supersize-me =allow`` for this (or maybe ``unstarving``?)  ;)

**Comment by ziceptor** [comment]
I want 6. Execution of arbitrary commands. For example turning off /fly. Etc.

**Comment by bhi-elliot** [comment]
14 would be amazing. Thanks for including it!

**Comment by zThana** [comment]
Still no updates on this?
